components/org.wso2.carbon.security.caas/src/main/java/org/wso2/carbon/security/caas/user/core/store/IdentityStore.java

```diff
@@ -184,17 +185,10 @@ void init(DomainManager domainManager, Map<String, IdentityStoreConnectorConfig>
     /**
      * Checks whether the user is in the group.
      *
-     * @param userId  Id of the user.
-     * @param groupId Id of the group.
+     * @param userName user name of the user.
+     * @param groupId  Id of the group.
      * @return True if the user is in the group.
      * @throws IdentityStoreException Identity Store Exception.
      */
-    boolean isUserInGroup(String userId, String groupId) throws IdentityStoreException;
-
-    /**
-     * Get all available identity store connector names.
-     *
-     * @return Map of display names against the connector id.
-     */
-    Map<String, String> getAllIdentityStoreNames();
+    boolean isUserInGroup(String userName, String groupId) throws IdentityStoreException;
```

Comment:
Users in identity store are resolved using a unique / primary attribute. This will be added as a TODO